src/map/skill.c

```diff
@@ -18881,7 +18881,7 @@ void skill_init_unit_layout (void)
 				}
 				break;
 			default:
-				skill->init_unit_layout_unknown(i);
+				skill->init_unit_layout_unknown(skill->dbs->db[i].nameid);
```

Comment:
probably better here send i as before.
but in function skill_init_unit_layout_unknown do like this:
```
ShowError("unknown unit layout at skill %d\n", skill->dbs->db[skill_idx].nameid);
```

Reply:
Done